built_value_generator/test/serializer_generator_test.dart

```diff
@@ -157,6 +157,27 @@ abstract class ValueBuilder implements Builder<Value, ValueBuilder> {
 '''), contains('implements StructuredSerializer<Value>'));
     });
 
+    test('handle \$ special character in class name', () async {
```

Comment:
Same here, please remove and instead make the classes added to the end to end tests have serializers.